sites/partners/src/components/shared/ExportTermsDialog.module.scss

```diff
@@ -14,3 +14,7 @@
   font-weight: var(--seeds-font-weight-semibold);
   font-family: var(--seeds-font-sans);
 }
+
+.terms-bold-text {
```

Comment:
This is a nit, but I think it looks a little subtle as is, and might lean toward using `--seeds-font-weight-bold`, but will absolutely let you use your UI discretion here

Reply:
Good point 🍎  just made that update!